cppwinrt/type_writers.h

```diff
@@ -44,12 +44,12 @@ namespace cppwinrt
         }
     };
 
-    bool operator==(type_name const& left, type_name const& right)
+    inline bool operator==(type_name const& left, type_name const& right)
```

Comment:
These don't strictly need to be inline because they're only ever included in a single TU. And again, static will issue a compiler warning if unused.

Reply:
For all the inlines? or just that one?